src/Forms/UrlField.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace SilverStripe\Forms;
+
+/**
+ * Text input field with validation for a url using filter_var($url, FILTER_VALIDATE_URL)
+ * The value of the field must start with a protocol such as http:// or https://
+ */
+class UrlField extends TextField
+{
+    protected $inputType = 'text';
+
+    public function Type()
+    {
+        return 'text';
```

Comment:
```suggestion
        return 'text url';
```
Gives people a css selector for this field type. Could be useful e.g. for adding custom front-end validation, different styling, etc.

Reply:
Removed

Reply:
I'm asking you to add it though? 😅

Reply:
Added